app/src/org/commcare/android/tasks/ManageKeyRecordTask.java

```diff
@@ -214,10 +214,9 @@ private void cleanupUserKeyRecords() {
                 }
                 
                 continue;
-            }
-            else if(record.getType() == UserKeyRecord.TYPE_NEW) {
+            } else if(record.getType() == UserKeyRecord.TYPE_NEW) {
                 //See if we have another sandbox with this ID that is fully initialized.
-                if(app.getStorage(UserKeyRecord.class).getIDsForValues(new String[] {UserKeyRecord.META_SANDBOX_ID, UserKeyRecord.META_KEY_STATUS}, new Object[] {record.getUuid(), UserKeyRecord.TYPE_NORMAL}).size() > 0) {
+                if(storage.getIDsForValues(new String[] {UserKeyRecord.META_SANDBOX_ID, UserKeyRecord.META_KEY_STATUS}, new Object[] {record.getUuid(), UserKeyRecord.TYPE_NORMAL}).size() > 0) {

```

Comment:
There isn't any problem with assigning app.getStorage(UserKeyRecord.class) to 'storage' and calling 'storage' multiple times, right?